Assignment/Assignment/SampleData.cs

```diff
@@ -1,30 +1,48 @@
-﻿using System;
-using System.Collections.Generic;
+﻿namespace Assignment;
 
-namespace Assignment
+public class SampleData : ISampleData
 {
-    public class SampleData : ISampleData
-    {
-        // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+    // 1.
+    private readonly Lazy<IEnumerable<string>> _LazyCsvRows = new Lazy<IEnumerable<string>>(
+        () => File.ReadLines(@"People.csv").Where(line => !string.IsNullOrWhiteSpace(line)).Skip(1));
+    public IEnumerable<string> CsvRows { get { return _LazyCsvRows.Value; } }
 
-        // 2.
-        public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows() 
-            => throw new NotImplementedException();
+    // 2.
+    public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows()
+    {
+        return CsvRows.Select(line => line.Split(',')).Select(x => x[6]).OrderBy(x => x).Distinct().ToList();
+    }
 
-        // 3.
-        public string GetAggregateSortedListOfStatesUsingCsvRows()
-            => throw new NotImplementedException();
+    // 3.
+    public string GetAggregateSortedListOfStatesUsingCsvRows()
+    {
+        string[] state = GetUniqueSortedListOfStatesGivenCsvRows().Select(x => x).ToArray();

```

Comment:
```suggestion
        string[] state = GetUniqueSortedListOfStatesGivenCsvRows().ToArray();

```
You don't need this here